config.cordovabuild.xml

```diff
@@ -112,6 +112,8 @@
     </plugin>
     <plugin name="edu.berkeley.eecs.emission.cordova.auth" spec="https://github.com/e-mission/cordova-jwt-auth.git" />
     <preference name="SplashShowOnlyFirstTime" value="false" />
+    <allow-navigation href="https://accessmap.io/*" />
+    <allow-navigation href="https://emission-accessmap.open-to-all.com/*" />
```

Comment:
I hope this is not required because `config.xml` is part of the native code and using channels can only apply native code changes.